Samples/azure-storage/Azure.Java.Fluent/implementation/StorageAccountInner.java

```diff
@@ -5,6 +5,7 @@
 
 import petstore.StorageAccountProperties;
 import com.microsoft.azure.Resource;
+import com.microsoft.azure.Resource;
```

Comment:
@jianghaolu you may want to look at why the import is repeated in the sample. This is not related to the change @vishrutshah is doing, he's just regenerating samples.